itest/lnd_test.go

```diff
@@ -151,9 +154,10 @@ func TestLightningNetworkDaemon(t *testing.T) {
 		}
 	}
 
-	height := harnessTest.CurrentHeight()
-	t.Logf("=========> tests finished for tranche: %v, tested %d "+
-		"cases, end height: %d\n", trancheIndex, len(testCases), height)
+	//nolint:forbidigo
+	fmt.Printf("=========> tranche %v finished, tested %d cases, mined "+
```

Comment:
Why do we switch from `t.Logf()` to `fmt.Printf()`? Just to get rid of the indentation? Or also so it's always printed, no matter if `-test.v` is specified?

Reply:
yeah mostly due to the `-test.v` thing, but also because we now log quite some info when turning on `-test.v` - my idea is to keep this print for a while until we are sure there are no more flakes and the shuffling works as expected. Then we can further hide this info from the CI (say after one or two releases).